pkg/commands/run_test.go

```diff
@@ -166,36 +260,15 @@ const expectedS3ScanResult = `{
             }
           }
         },
-        {
-          "Type": "AWS",
-          "ID": "AVD-AWS-0092",
-          "AVDID": "AVD-AWS-0092",
-          "Title": "S3 Buckets not publicly accessible through ACL.",
-          "Description": "Buckets should not have ACLs that allow public access",
-          "Resolution": "Don't use canned ACLs or switch to private acl",
-          "Severity": "HIGH",
-          "PrimaryURL": "https://avd.aquasec.com/misconfig/avd-aws-0092",
-          "References": [
-            "https://avd.aquasec.com/misconfig/avd-aws-0092"
-          ],
-          "Status": "PASS",
-          "Layer": {},
-          "CauseMetadata": {
-            "Resource": "arn:aws:s3:::examplebucket",
-            "Provider": "aws",
-            "Service": "s3",
```

Comment:
This check is still considered [passed](https://github.com/aquasecurity/trivy-aws/pull/263/files#diff-ef103af943920a01839a075b421bdcff3319c58012056e739e5b3d048ae8389fR50-R71), but due to the specificity of Rego checks, all passed checks will have an empty target.